x-pack/plugins/lists/common/schemas/request/read_exception_list_schema.ts

```diff
@@ -11,11 +11,13 @@ import * as t from 'io-ts';
 import { NamespaceType, id, list_id, namespace_type } from '../common/schemas';
 import { RequiredKeepUndefined } from '../../types';
 
-export const readExceptionListSchema = t.partial({
-  id,
-  list_id,
-  namespace_type, // defaults to 'single' if not set during decode
-});
+export const readExceptionListSchema = t.exact(
```

Comment:
NOTE: Added, as I believe we're going for adding this to all our schemas?

Reply:
Correct. You have to add t.exact( otherwise extra keys and value will be allowed and the exact check will not be capable of working. So anything missing an t.exact should be consider a critical bug that we need to fix.

Thank you for this fix.